eng/tools/TypeSpecValidation/src/utils.ts

```diff
@@ -6,7 +6,7 @@ export async function runCmd(cmd: string, cwd: string) {
   const { err, stdout, stderr } = (await new Promise((res) =>
     exec(
       cmd,
-      { encoding: "utf8", maxBuffer: 1024 * 1024 * 64, cwd: cwd },
+      { encoding: "utf8", maxBuffer: 1024 * 1024 * 64, cwd: cwd, shell: "powershell.exe" },
```

Comment:
```suggestion
      { encoding: "utf8", maxBuffer: 1024 * 1024 * 64, cwd: cwd },
```

Revert to remove dependency on `pwsh`